torch/_inductor/scheduler.py

```diff
@@ -922,6 +922,14 @@ def apply_new_loop_order(self, new_order: Sequence[int]) -> None:
 
         self.refresh_dependencies(normalize=False)
 
+        from .codegen.simd import SIMDScheduling
+
+        # TODO(shunting) if this cause compilation time increase when
+        # enabling LOAF by default, try just clearing the specific cache
+        # entry by using a customized cache implemetation rather than
+        # lru_cache.
+        SIMDScheduling.candidate_tilings.cache_clear()
```

Comment:
Would be easy enough to roll our own here - see, this example found from googling if there is a remove method on lru_cache - 

https://pastebin.com/LDwMwtp8